### PR TITLE
add Linux option to pack_iso

### PR DESCRIPTION
**Pull request by deinarso**
This script wont work on linux with out such an option ;) Thanks.

### CODE REVIEW

**Comment by chorrell** [comment]
LGTM

File: bin/build-image [hunk 1]
```diff
@@ -53,6 +53,10 @@ pack_iso() {
       mkisofs -J -R -input-charset iso8859-1 -V "SDC Guest Tools" \
         -o sdc-vm-tools-$timestamp.iso sdc-vm-tools
       ;;
+     Linux)
```
Comment (chorrell) [inline bin/build-image]: Can you fix the tab/spacing here so it matches Darwin, SunOS etc.?

**Comment by deinarso** [comment]
Sorry about that, it had looked good on my screen.  Fixed

**Comment by chorrell** [comment]
Great thanks!

**Pull request merged**